jaas/modules/src/main/java/org/apache/karaf/jaas/modules/properties/PropertiesBackingEngine.java

```diff
@@ -52,14 +52,13 @@ public void addUser(String username, String password) {
         if (username.startsWith(GROUP_PREFIX))
             throw new IllegalArgumentException("Prefix not permitted: " + GROUP_PREFIX);
 
-        addUserInternal(username, password);
+        addUserInternal(username, encryptionSupport.encrypt(password));
```

Comment:
Why forcing encryption here ? It's an optional feature.

Reply:
I just moved this call from its original location in `addUserInternal`:
https://github.com/apache/karaf/blob/e9b9c973569596c8931cc4e8f7d62744d9c3ede5/jaas/modules/src/main/java/org/apache/karaf/jaas/modules/properties/PropertiesBackingEngine.java#L62

The call is only needed when adding a user and shouldn't be made when adding a group. There is the risk of encrypting `""`, which with the defaults, results in `{CRYPT}ABC...{CRYPT}`. After `jaas:group-add karaf newGrup`, 
`jaas:user-list` will return 
```
User Name | Group   | Role
----------+---------+-------------------------------------------------------------------------------
karaf     | newGrup | {CRYPT}ABC...{CRYPT}
```

Maybe at this point, we should create another `addUserInternal` with just the `username` as an argument: `private void addUserInternal(String username)`, or another function just for groups...